include/crow/http_server.h

```diff
@@ -161,9 +161,21 @@ namespace crow
 
         void stop()
         {
-            io_service_.stop();
+            should_close_ = false;                                                 //Prevent the acceptor from taking new connections
```

Comment:
Is it used somewhere?

Reply:
I must have missed adding functionality to this when I was first working on the feature. The comment seems to indicate that setting this to `true` would disable `do_accept()`. I'm pushing a commit to get that working now.